simde/x86/svml.h

```diff
@@ -3700,10 +3754,16 @@ simde_mm256_erfc_pd (simde__m256d a) {
       r_,
       a_ = simde__m256d_to_private(a);
 
-    SIMDE_VECTORIZE
-    for (size_t i = 0 ; i < (sizeof(r_.f64) / sizeof(r_.f64[0])) ; i++) {
-      r_.f64[i] = simde_math_erfc(a_.f64[i]);
-    }
+    #if SIMDE_NATURAL_VECTOR_SIZE_LE(128)
+      for (size_t i = 0 ; i < (sizeof(r_.m128d) / sizeof(r_.m128d[0])) ; i++) {
+        r_.m128d[i] = simde_mm_erfc_pd(a_.m128d[i]);
+      }
+    #else
+      SIMDE_VECTORIZE
+      for (size_t i = 0 ; i < (sizeof(r_.f64) / sizeof(r_.f64[0])) ; i++) {
+        r_.f64[i] = simde_math_erfc(a_.f64[i]);
+      }
+    #endif
```

Comment:
This was already merged to the main development branch in https://github.com/simd-everywhere/simde/commit/4400413b273525ef4daf642ee34bf97a9d2caf28#diff-b5a78fcbe9bbb05b9cdc4eb3d453fce84b73becf0b4ed4ca2ca677f655ca5539R3709 but it went with `256` as the value of `SIMDE_NATURAL_VECTOR_SIZE_LE` ; when I merged it in I kept the `256`, but please double check that, thanks!